src/components/SamplePaper.tsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import Paper from "@material-ui/core/Paper";
+
+const SamplePaper: React.FC = () => {
+  return (
+    <Paper elevation={1}>
+      <h1>Sample Page</h1>
+    </Paper>
```

Comment:
I think there is no need to bring the sample card back. We can also start over with components that are actually useful